tests/resources.py

```diff
@@ -20,6 +20,10 @@ def find(name: str) -> str:
     return os.path.normpath(os.path.join(os.path.dirname(__file__), DATA_DIR, name))
 
 
+def find_in_testdata(name: str) -> Path:
+    return (Path(__file__).parent / "testdata" / name).resolve()
```

Comment:
Why is the new test file the only one "loose" outside testdata/python3/data? That makes this new lookup unnecessary.

Reply:
I think it makes sense to try and move data in `testdata` instead of `python3`. I tried moving everything some time ago but a lof of tests are dependent on the current directory structure. That said, I think adding new data in `testdata` instead of `testdata/python3` makes more sense. I'll happily move if you want to keep using `python3` though.

Reply:
Yeah I'm -0 on moving things around, I just don't see the point. And I'm -0.83333 on having tests in two different places.

Reply:
Changed to be in one place.